src/include/kdump/dialogs.rb

```diff
@@ -531,7 +531,7 @@ def kdump_memory_widget
       else
         _("Kdump Memor&y [MiB]")
       end
-      low_range = format(_("(%{min} - %{max}, default: %{default})"),
+      low_range = format(_("(min:%{min}; max:%{max}; suggested: %{default})"),
```

Comment:
There is no space after "min:" or "max:", but there is a space after "suggested:". Looks inconsistent.

BTW, maybe we should add a comment for the translators to make sure they don't translate the expressions between curly brackets. Since this syntax is ruby-specific, they tend to get it wrong.